exchanges/bitfinex/bitfinex.go

```diff
@@ -491,6 +494,72 @@ func (b *Bitfinex) GetSiteListConfigData(ctx context.Context, set string) ([]str
 	return resp[0], nil
 }
 
+// GetSiteInfoConfigData returns site configuration data by pub:info:{AssetType} as a map
+// path should be bitfinexInfoPairs or bitfinexInfoPairsFuture???
+// NOTE: See https://docs.bitfinex.com/reference/rest-public-conf
+func (b *Bitfinex) GetSiteInfoConfigData(ctx context.Context, assetType asset.Item) (map[currency.Pair]order.MinMaxLevel, error) {
+	var path string
+	switch assetType {
+	case asset.Spot:
+		path = bitfinexInfoPairs
+	case asset.Futures:
+		path = bitfinexInfoFuturePairs
+	default:
+		return nil, fmt.Errorf("invalid asset type for GetSiteInfoConfigData: %s", assetType)
+	}
+	url := bitfinexAPIVersion2 + path
+	var resp [][][]any
+
+	err := b.SendHTTPRequest(ctx, exchange.RestSpot, url, &resp, status)
+	if err != nil {
+		return nil, err
+	}
+	if len(resp) != 1 {
+		return nil, errors.New("response did not contain only one item")
+	}
+	pairs := map[currency.Pair]order.MinMaxLevel{}
+	data := resp[0]
+	for i := range data {
+		pairSymbol, ok := data[i][0].(string)
+		if len(data[i]) != 2 {
+			return nil, errors.New("response contained a tuple without exactly 2 items")
+		}
```

Comment:
This needs to be moved above `pairSymbol, ok := data[i][0].(string)` then we can hug the ok check with each other as well.

Reply:
done